Answers.md

```diff
@@ -1 +1,17 @@
+1. Describe the biggest difference between .forEach & .map.
 
+.forEach opperate exactly like a for loop itterating through each element and modifying, comparing, or returning a value.
+
+.map takes and array and returns a brand new array.
```

Comment:
what do you mean by "takes"?

Reply:
whoops didn't that's completely incorrect. .map is an array method that modifies the array depending on the instructions you give it. So if you have a number array and you tell .map to multiple each number by two it will return a new array of those numbers multiplied by two without modifying the original array.